solr/core/src/test/org/apache/solr/uninverting/TestFieldCacheVsDocValues.java

```diff
@@ -619,6 +619,6 @@ private void assertEquals(long numOrds, TermsEnum expected, TermsEnum actual) th
   
   protected boolean codecAcceptsHugeBinaryValues(String field) {
```

Comment:
do we still need this method?

Reply:
Right, we don't. I actually removed larger conditional blocks of code after this method is gone.